deploy/katello.spec

```diff
@@ -148,6 +167,24 @@ getent group katello-shared > /dev/null || groupadd -r katello-shared
 usermod -a -G katello-shared katello
 exit 0
 
+# ------ SAM ------------------
+
+%package sam
+Summary: Package that installs only the Subscription and basic Content Management parts of Katello
+Group:  Applications/System
+
+# Require the common package and ensure the katello-sam package 
+# can't be installed on the same system as katello
+Requires:       %{name}-common = %{version}-%{release}
+Conflicts:      %{name}
+Requires:       rubygem-hammer_cli_sam
+Requires:       ruby193-rubygem-foreman_sam
```

Comment:
yeah, we don't deploy in non-SCL but it is safer to always be clear about this in the specfile